server/app/rest_api/rest.py

```diff
@@ -201,6 +201,53 @@ def get(self):
         else:
             return make_response(jsonify(annotation_response))
 
+    @swagger.doc({
+        "summary": "Fetch annotations (metadata) for filtered subset.",
+        "tags": ["annotations"],
+        "parameters": [
+            {
+                "in": "query",
+                "name": "annotation-name",
+                "type": "string",
+                "description": "comma-separated annotation keys, ex: num_genes,percent_mito"
```

Comment:
Description is obsolete - we now take a normal query string, rather than a comma-separated list.

Reply:
fixed